examples/swm_bickley_jet.jl

```diff
@@ -26,34 +36,28 @@ grid = RegularCartesianGrid(
     topology=(Periodic, Bounded, Bounded)
     )
 
+
 model = ShallowWaterModel(
-    grid=grid,
-    gravitational_acceleration=1,
     architecture=CPU(),
-    coriolis=FPlane(f=1),
-    advection=WENO5()
+ #   timestepper = :RungeKutta3, 
+    advection=WENO5(),
+    grid=grid,
+    gravitational_acceleration=g,
+    coriolis=FPlane(f=f)
     )
 
-### Parameters
-Δη = 0.1
-g  = model.gravitational_acceleration
-f  = model.coriolis.f
-
-k   = 4
-ℓ   = 1
-amp = 0e-3
-
 ### Basic State
-H₀(x, y, z) =   1.0
-H(x, y, z)  =   H₀(x, y, z) - Δη * tanh(y)
-U(x, y, z)  =   g / f * Δη * sech(y)^2
+H₀(x, y, z) =   10.0
+H(x, y, z)  =   H₀(x, y, z) - f / g * Δη * tanh(y) + f / g * 2 * y / Ly
+U(x, y, z)  =   Δη * sech(y)^2 - 2 / Ly
 UH(x, y, z) = U(x, y, z) * H(x, y, z)
-Ω( x, y, z) =  2 * g / f * Δη * sech(y)^2 * tanh(y)
+Ω( x, y, z) =  2 * Δη * sech(y)^2 * tanh(y)
 
 ### Perturbation
-h_perturbation( x, y, z) = amp * exp(-y^2 / 2ℓ^2) * cos(k * x * 2 * π / Lx )
-uh_perturbation(x, y, z) = amp * exp(-y^2 / 2ℓ^2) * cos(k * x * 2 * π / Lx )
-vh_perturbation(x, y, z) = amp * exp(-y^2 / 2ℓ^2) * cos(k * x * 2 * π / Lx )
+ψ(x, y, z, ℓ , k) = exp(-(y + ℓ/10)^2 / 2ℓ^2) * cos(k * x) * cos(k * y)
+h_perturbation( x, y, z) = amp * ψ(x, y, z, ℓ, k)
+uh_perturbation(x, y, z) = amp * ψ(x, y, z, ℓ, k)
+vh_perturbation(x, y, z) = amp * ψ(x, y, z, ℓ, k)
 
 ## Total fields
 uh(x, y, z) = UH(x, y, z) + uh_perturbation(x, y, z)
```

Comment:
In many of the other examples we use notation like `uh_0` or `uh_i` to refer to the "initial" `uh`, and reserve `uh` for the field (eg `uh = model.solution.uh`). Can we do the same here?

Reply:
Very good idea.  I will make that change here and try and keep this notation in the future.  It's much clearer!

Thanks also for pointing me to the code.  This is not the geostrophically balanced Bickley jet that I know but it should do the trick.  After the code has been validated, I will return to the balanced Bickley jet.  Now to get something working!